app/bundles/CoreBundle/Helper/ArrayHelper.php

```diff
@@ -94,6 +94,27 @@ function ($value) {
         );
     }
 
+    /**
+     * Flip array or sub arrays.
+     *
+     * @param array<int|string|array<int|string>> $masterArrays
+     *
+     * @return array<int|string|array<int|string>>
+     */
+    public static function flipArray(array $masterArrays): array
+    {
+        if (!is_array(end($masterArrays))) {
+            return array_flip($masterArrays);
+        }
+
+        return array_map(
+            function (array $subArray) {
+                return array_flip($subArray);
+            },
+            $masterArrays
+        );
```

Comment:
We can use arrow functions now 🎉 
```suggestion
        return array_map(
            fn (array $subArray) => array_flip($subArray),
            $masterArrays
        );
```